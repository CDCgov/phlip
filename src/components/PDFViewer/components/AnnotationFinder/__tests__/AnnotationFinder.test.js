import React from 'react'
import { AnnotationFinder } from '../index'
import { shallow } from 'enzyme'

const props = {
  count: 4,
  current: 0,
  users: [],
  handleNext: jest.fn(),
  handlePrevious: jest.fn()
}

describe('PDFViewer - AnnotationFinder', () => {
  test('should render correctly', () => {
    expect(shallow(<AnnotationFinder {...props} />)).toMatchSnapshot()
  })
})
