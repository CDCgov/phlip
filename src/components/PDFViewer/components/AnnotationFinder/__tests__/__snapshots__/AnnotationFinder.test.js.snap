// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PDFViewer - AnnotationFinder should render correctly 1`] = `
<FlexGrid
  align="center"
  circular={false}
  container={true}
  flex={false}
  justify="flex-end"
  padding="0 8px"
  raised={true}
  style={
    Object {
      "alignSelf": "flex-end",
      "background": "#0000008a",
      "backgroundColor": "black",
      "height": 40,
      "position": "absolute",
      "right": 20,
      "top": 10,
      "width": "40%",
      "zIndex": 200,
    }
  }
  type="row"
>
  <FlexGrid
    align="center"
    circular={false}
    container={true}
    flex={true}
    justify="stretch"
    padding="0 10px 0 0"
    raised={false}
    style={Object {}}
    type="row"
  />
  <FlexGrid
    align="stretch"
    circular={false}
    container={false}
    flex={false}
    justify="stretch"
    padding={0}
    raised={false}
    style={
      Object {
        "marginRight": 15,
      }
    }
    type="column"
  >
    <WithStyles(Typography)
      style={
        Object {
          "color": "white",
        }
      }
    >
      1
      /
      4
    </WithStyles(Typography)>
  </FlexGrid>
  <FlexGrid
    align="stretch"
    circular={false}
    container={false}
    flex={false}
    justify="stretch"
    padding={0}
    raised={false}
    style={
      Object {
        "borderLeft": "2px solid white",
        "height": "50%",
      }
    }
    type="column"
  />
  <FlexGrid
    align="stretch"
    circular={false}
    container={false}
    flex={false}
    justify="stretch"
    padding={0}
    raised={false}
    style={
      Object {
        "marginLeft": 8,
      }
    }
    type="column"
  >
    <IconButton
      color="rgba(0, 0, 0, 0.54)"
      disabled={true}
      iconSize={24}
      onClick={[Function]}
      tooltipText=""
    >
      keyboard_arrow_up
    </IconButton>
    <IconButton
      color="white"
      disabled={false}
      iconSize={24}
      onClick={[Function]}
      tooltipText=""
    >
      keyboard_arrow_down
    </IconButton>
  </FlexGrid>
</FlexGrid>
`;
